IRP/src/CommonModules/Localization/Module.bsl

```diff
@@ -1237,6 +1250,8 @@ Function Strings(Lang) Export
 	Strings.Insert("QuestionToUser_026", NStr("en = 'Do you want to update payment agent?'", Lang));
 	Strings.Insert("QuestionToUser_027", NStr("en = 'Filled data by employee [%1] will be cleared. Continue?'", Lang));
 	Strings.Insert("QuestionToUser_028", NStr("en = 'Do you want to refund the client?'", Lang));
+	Strings.Insert("QuestionToUser_029", NStr("en = 'New document movements are no different from manual corrections. The ""Manual movements edit"" checkbox can be removed. Remove check-box?'", Lang));
```

Comment:
"New document movements match the manual corrections, making the "Manual Movements Edit" checkbox unnecessary. Would you like to remove this checkbox?"